lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: get_certificate
+author: "John Westcott IV"
+version_added: "2.7"
+short_description: Get a certificate from a host:port
+description:
+    - Makes an SSL connection and returns information about the presented certificate
+options:
+    host:
+      description:
+        - The host to get the cert for (IP is fine)
+      required: True
+    ca_certs:
+      description:
+        - A file containing a list of root certificates; if present, the cert will be validated against these root certs.
+      required: False
+    port:
+      description:
+        - The port to connect to
+      required: True
+    timeout:
+      description:
+        - The timeout in seconds
+      required: False
+      default: 10
+
+notes:
+  - When using ca_certs on OS X it has been reported that in some conditions the validate will always suceed.
+
+requirements:
+  - "python >= 2.6"
+  - "ssl"
+  - "python-pyOpenSSL >= 0.15"
+  - "socket"
```

Comment:
This one too is a default module, right?

Reply:
Also Removed.